components/_patterns/02-molecules/menus/_menu-item.twig

```diff
@@ -1,10 +1,4 @@
-
-
-{% if not item_modifiers %}
```

Comment:
I admit this is ugly, but I believe that I added it so that we could pass item_modifiers to the template from another template. But, if we did not, then I had to initiate the array so that it could be added to. If we are no longer passing any item_modifiers (classes, really) then this is cool, but should be confirmed.

Reply:
Actually, this file isn't used anywhere. I created this originally as a part of my work. But I couldn't for the life of me figure out how to get the mega menu items to show using these templates, so I went back to just using the ones you created in the Drupal templates folder.

Your original work is in `/02-molecules/02-menu` my new work is in `/02-molecules/menu` (There's really no need to index them unless one thing depends on another. And that really only happens in the base folder... maybe atoms too.)

Reply:
If we do end up figuring out a way to use these though, that would be good. And in that case, it's actually best to re-set them here like the update does. If not, the `level-x`, `active`, etc. modifiers get passed down to children whether or not they apply to those children.

It was producing things like `class="main-menu__item--level-0 main-menu__item--level-1 main-menu__item--level-2` on a single item, which doesn't make sense. It should only have one level.

Anyway. This isn't used right now, so we can discuss it later if needed.